docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -1295,9 +1295,9 @@ push_circuit_breaker:
   # CLI flag: -ingester.push-circuit-breaker.cooldown-period
   [cooldown_period: <duration> | default = 10s]
 
-  # (experimental) How long the circuit breaker should wait between an
-  # activation request and becoming effectively active. During that time both
-  # failures and successes will not be counted.
+  # (experimental) How long after an initial request an activated circuit
+  # breaker should wait before becoming effectively active. During this time
+  # neither failures nor successes will not be counted.
```

Comment:
```suggestion
  # (experimental) Duration, in seconds, after an initial request an activated circuit
  # breaker should wait before becoming effectively active. During this time,
  # neither failures nor successes are counted.
```

Reply:
addressed in https://github.com/grafana/mimir/pull/9842/commits/24a3b0db894298d97e53aab419553c6d71074ec9